xml/System.Threading.Tasks/Task.xml

```diff
@@ -734,7 +734,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This property returns a task whose <xref:System.Threading.Tasks.Task.Status%2A> property is set to <xref:System.Threading.Tasks.TaskStatus.RanToCompletion>. To create a task that returns a value and runs to completion, call the <xref:System.Threading.Tasks.Task.FromResult%2A> method.  
+ This property returns a task whose <xref:System.Threading.Tasks.Task.Status%2A> property is set to <xref:System.Threading.Tasks.TaskStatus.RanToCompletion>. To create an equivalent with a result, call the <xref:System.Threading.Tasks.Task.FromResult%2A> method.  
```

Comment:
For me the former setence was clearer. "Equivalent" is quite generic, and equivalent of what? "with a result" is got by FromResult too.

Maybe just 
```suggestion
 This property returns a task whose <xref:System.Threading.Tasks.Task.Status%2A> property is set to <xref:System.Threading.Tasks.TaskStatus.RanToCompletion> direclty, i.e. the task didn't run. To create a task that returns a value and runs to completion, call the <xref:System.Threading.Tasks.Task.FromResult%2A> method.  
```